ui-sdk-components-sample/UISDK components sample/UIKit Karhoo Component/ViewController.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  ViewController.swift
+//  UIKit Karhoo Component
+//
+//  Copyright © 2020 Karhoo. All rights reserved.
+//
+
+import UIKit
+import KarhooUISDK
+import KarhooSDK
+import SwiftSpinner
+
+protocol SampleView {
+    func setUpView()
+    func quoteListHidden(_ hidden: Bool)
+    func dismissTopViewController()
+    func presentView(viewController: UIViewController)
+}
+
+class ViewController: UIViewController {
+
+    let presenter: ViewControllerPresenter = ViewControllerPresenter()
+
+    // addressBar: Allows user to set booking details
+    private lazy var addressBar: AddressBarView = {
+        let addressBar = KarhooUI.components.addressBar(journeyInfo: nil)
+        addressBar.translatesAutoresizingMaskIntoConstraints = false
+        return addressBar
+    }()
+
+    // A scrollable list of quotes based on the set booking details
+    private lazy var quoteList: QuoteListView = {
+        let quoteList = KarhooUI.components.quoteList()
+        quoteList.set(quoteListActions: self)
```

Comment:
Do we have something similar for the AddressBarView? to handle the output of the AddressBarView

Reply:
No output on the AddressBarView - it updates BookingStatus observers which are used in all components